src/lib/messages.js

```diff
@@ -12,11 +12,14 @@ export const introMessage = [
 	"| CheerpX includes an x86-to-WebAssembly JIT compiler, a virtual block-based  |",
 	"| file system, and a Linux syscall emulator                                   |",
 	"|                                                                             |",
-	"| [News] CheerpX 1.0 officially released!                                     |",
+	"| [News] Our Hackathon is coming! check out the site for updates!             |",
 	"|                                                                             |",
-	"| " + underline + "https://cheerpx.io/blog/cx-10" + normal + "                                               |",
+	"| " + underline + "https://cheerpx.io/hackathon" + normal + "                                                |",
 	"|                                                                             |",
-	"| Try out the new Alpine / Xorg / i3 WebVM: " + underline + "https://webvm.io/alpine.html" + normal + "      |",
```

Comment:
Please split this change in two commits. I want to have a simple way of restoring the alpine link with a single revert after the AMA is done.